app/streaming/session.cpp

```diff
@@ -362,8 +369,24 @@ bool Session::initialize()
         return false;
     }
 
+#ifdef HAVE_EGL
+    /* Request opengl ES 3.0 context for EGL Renderer, otherwise it will
+     * SIGSEGV: https://gitlab.freedesktop.org/mesa/mesa/issues/1011
+     */
+    SDL_SetHint(SDL_HINT_OPENGL_ES_DRIVER, "1");
```

Comment:
Using `SDL_CreateRenderer()` you can explicitly specify your renderer driver without having to set a hint.

Reply:
Hum it seems that I need to keep the hint on X11, otherwise the `opengles2` renderer is not available (I can put it inside my `initialize()` method tho)